### DISCUSSION TITLE
Using konsta with plain html/js + tailwind (no framework)

### DISCUSSION THREAD

**Discussion by kosirm** [konstaui/konsta#189] [Q&A]
I would like to build website with plain html/javascript and use this library. I'm using tailwind. Is it possible to use konsta with NO framework (vue, react, svelte), just plain html and js? Thank you for any hint.

**Comment by vitalijalbu** [comment]
> I would like to build website with plain html/javascript and use this library. I'm using tailwind. Is it possible to use konsta with NO framework (vue, react, svelte), just plain html and js? Thank you for any hint.

Hi, did u find a solution?
